gwswlib/importer.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import logging
+import os
+import csv
+
+from gwswlib.hydx import Hydx, ConnectionNode
+
+logger = logging.getLogger(__name__)
+
+
+def importhydx(hydxpath):
+    """Read set of hydx-csvfiles and return Hydx objects"""
+    print("hoi")
+
+    hydx = Hydx()
+
+    # check if hydx exists
```

Comment:
Het is niet "hydx" maar een specifieke file.
Èn de code is duidelijk genoeg, dus die hoef je niet uit te leggen.